scripts/setup/debians/build_cmake.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Copyright (c) 2017, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+#
+# All rights reserved.
+#
+# The Astrobee platform is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
```

Comment:
This version of cmake is broken for armhf systems. We need cmake to compile our custom debians. I added the patched debians to our new repo. I didn't have a better place to put the script so I figured this was good.

Reply:
In case it wasn't clear, we need cmake for the debbuild multistrap config that we use to compile our custom debians.

Reply:
It looks like the patch is just adding 

add_definitions(-D_FILE_OFFSET_BITS=64)

Can't we do this in our cmakelists without modifying cmake?

Reply:
I haven't tried that but I'm not very optimistic it would work. I believe the problem is more internal to cmake (handling files I think). BTW, this patch has been made official in the latest versions of cmake.